//
// Copyright 2018 Asylo authors
//
// Licensed under the Apache License, Version 2.0 (the "License");
// you may not use this file except in compliance with the License.
// You may obtain a copy of the License at
//
//     http://www.apache.org/licenses/LICENSE-2.0
//
// Unless required by applicable law or agreed to in writing, software
// distributed under the License is distributed on an "AS IS" BASIS,
// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
// See the License for the specific language governing permissions and
// limitations under the License.
//

syntax = "proto2";

package asylo;

import "asylo/enclave.proto";

// Configuration for an Fortanix EDP enclave loader.
message FortanixEdpLoadConfig {
  // Path to the enclave binary (.sgxs) file to load.
  optional string enclave_path = 1;
}

extend EnclaveLoadConfig {
  optional FortanixEdpLoadConfig fortanix_edp_load_config = 263000382;
}
